documentation/troubleshooting.md

```diff
@@ -251,3 +251,11 @@ To change your script execution policy:
     documentation about [execution
     policies](http://go.microsoft.com/fwlink/?LinkID=135170) and [how to set
     them](https://msdn.microsoft.com/en-us/powershell/reference/5.1/microsoft.powershell.security/set-executionpolicy).
+
+## Getting error 'Could not parse PKey: no start line' using SSH private key
+
+This error indicates that the SSH private key provided is encrypted, which is unsupported by Bolt
+if the key is provided under `key-data`. You can manually decrypt the key by running `openssl rsa -in
+<key_file>` and providing your passphrase, if providing a decrypted key is feasible for your use
+case and security practices. You can also provide a path to the key instead of `key-data` and add it
```

Comment:
Adding it to SSH agent is the part that makes this work. Providing a path is unnecessary if you're using the SSH agent and _doesn't_ work if you aren't.

Reply:
Ah good point!